plenum/test/client/test_client_request_nack.py

```diff
@@ -30,6 +30,7 @@ def request1(wallet1):
     return req
 
 
+@pytest.mark.skip(reason="old style plugin")
```

Comment:
Do we have another test to check NACKs?

Reply:
This test was not checking NACK, it is checking rejection by old style plugin

Reply:
Ok, then should we remove this test completely?